### ISSUE TITLE
Switching Install On Me

### ISSUE THREAD

**Issue by PreachJohn** [sindresorhus/caprine#1920]
I really don't appreciate that when I try to install 255.6 to fix issues, 256.1 force installs instead. 

---

Caprine 2.56.1
Electron 20.0.3
win32 6.1.7601
Locale: en-US

**Comment by dusansimic** [comment]
What issues are you trying to fix by installing 2.55.6? Also, the latest version is probably being installed b ecause auto updates are enabled.

**Comment by PreachJohn** [comment]
You know the issues with 256.1.
3 off the top of my head:
Freezing too often.
If the little 'Messenger' box at the very top left corner shows a
Message(s) I have to Relaunch Caprine to access it most times. I am often
Relaunching over and over.
And Delete Chat doesn't work.
Thanks for the tip re: Auto Updates. I will try it.

**Comment by dusansimic** [comment]
> And Delete Chat doesn't work.

This won't be fixed by installing an older version because of #1906

**Comment by dusansimic** [comment]
How did you remove that sidebar? Just installed an older version of Caprine? I'm asking because that shouldn't be possible since we don't influence that sidebar in any way, we just display the messenger website.

**Comment by dusansimic** [comment]
Closing since it's solved

**Comment by PreachJohn** [comment]
After your suggestion to uncheck Auto Updates before I uninstalled 256.1, I
successfully installed 255.6 as it turned out for only for a short while
again.
Prior to the unchecking of Auto Update before 255.6 would always install as
256.1  I did periodically check my version if 255.6.
Then on one particular loading of 255.6 Caprine from the System Tray Icon,
Set Up appeared out of nowhere; back to 256.1 that I don't want.
So unchecking in 256.1 again before uninstall, I installed 255.6. It only
always installs as 256.1 again.
